libraries/chain/include/eosio/chain/wasm_interface_private.hpp

```diff
@@ -197,7 +198,7 @@ struct eosvmoc_tier {
          }
       }
 
-      sync_call_return_code do_sync_call( const digest_type& code_hash, const uint8_t& vm_type, const uint8_t& vm_version, apply_context& context ) {
+      sync_call_return_code do_sync_call( const digest_type& code_hash, const uint8_t& vm_type, const uint8_t& vm_version, sync_call_context& context ) {
          return get_instantiated_module(code_hash, vm_type, vm_version, context)->do_sync_call(context);
```

Comment:
Am I missing something? Do sync calls not eosvmoc tierup?

Reply:
Ultimately they will but nothing OC is plumbed through yet. Some stuff (outside OC) that we're ignoring right now is going to need to be refactored for that.